vllm/worker/model_runner.py

```diff
@@ -682,6 +699,8 @@ def _prepare_model_input_tensors(
         slot_mapping_tensor = torch.tensor(slot_mapping,
                                            dtype=torch.long,
                                            device=self.device)
+        prefill_original_seq_lens_tensor = torch.tensor(
```

Comment:
Is it possible to push the creation of this tensor into `attn_backend.make_metadata`?